---
float64[] position
float64[] velocity
float64[] target_position
float64[] orientation
float64 reward